### PR TITLE
[textanalytics] merge single/multi category result

### PR DESCRIPTION
**Pull request by kristapratico**
Part of https://github.com/Azure/azure-sdk-for-python/issues/24769

This service version merges these two models into one. We will likely merge the input as well in another PR.

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 - Removed the Extractive Text Summarization feature and related models: `ExtractSummaryAction`, `ExtractSummaryResult`, and `SummarySentence`. To access this beta feature, install the `5.2.0b4` version of the client library.
+- `SingleCategoryClassifyResult` and `MultiCategoryClassifyResult` models have been merged into one model: `ClassifyDocumentResult`.
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:10]: Open to naming suggestions on this

Comment (iscai-msft) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:10]: I like the name ❤️

**Review by iscai-msft — APPROVED** [review]
wonderful work, as always

**Review by deyaaeldeen — APPROVED** [review]

**Pull request merged**